nautobot/docs/user-guide/administration/upgrading/from-v1/upgrading-from-nautobot-v1.md

```diff
@@ -36,6 +36,10 @@ All pre-migration checks passed.
 - Nautobot no longer uses or supports the use of `django-mptt`.
 - Nautobot no longer uses or supports the use of `django-rq`.
 
+## App Developer Interface
+
+All imports for app developers should now be sourced from `nautobot.apps.*` or provided models, as this is the supported public interface. This will help ensure proper SemVer is kept in tact and provide a place to add new features without breaking existing integrations.
```

Comment:
> I'm not sure we're at a point yet where we can dogmatically assert this

This seems like table stakes for 2.0, no?

Reply:
Perhaps it should be, but it currently isn't being tracked as such, will likely be a series of incremental improvements throughout 2.x.

Reply:
@lampwins I think we should have this for the 2.0 migration, otherwise we will be doing this again when we want to do the same thing and can't update anything until 3.0 again, plus it makes it should be easier for the app developers to know where to go to find things.

Reply:
Addressed in #4533